components/pluralrules/examples/elevator_floors.rs

```diff
@@ -0,0 +1,37 @@
+// An example application which uses icu_pluralrules to construct a correct
+// sentence for English based on the numerical value in Ordinal category.
+use icu_fs_data_provider::FsDataProvider;
+use icu_locale::LanguageIdentifier;
+use icu_pluralrules::{PluralCategory, PluralRuleType, PluralRules};
+
+const VALUES: &[usize] = &[0, 2, 25, 1, 3, 2, 4, 10, 7, 0];
+
+fn print(_input: &str, _value: Option<usize>) {
+    #[cfg(debug_assertions)]
+    if let Some(value) = _value {
+        println!("{}", _input.replace("{}", &value.to_string()));
+    } else {
+        println!("{}", _input);
+    }
+}
+
+fn main() {
+    let langid: LanguageIdentifier = "en".parse().expect("Failed to parse Language Identifier.");
+    let dtp = FsDataProvider::try_new("./tests/data/json_plurals_37")
```

Comment:
I don't think we want FsDataProvider in the PluralRules example.  It will bloat the binary and hide real changes that come from PluralRules.  However, we could keep it here for now and swap it out for StaticDataProvider (#78) that will have much smaller code size.

Reply:
I'm torn on this. `FsDataProvider` is the real-world scenario, and if we use `Invariant`, then we "hide" the impact of `FsDataProvider` changes on real world scenario.

Reply:
But yes, `Static` may be a better choice!

Reply:
My approval stands.  We can swap it out for StaticDataProvider when it is ready.